src/AmazonSES.php

```diff
@@ -18,6 +18,9 @@ class AmazonSES implements EmailSenderInterface
     protected $host;
     protected $logger;
 
+    public $verifyPeer = true;
+	public $verifyHost = true;
+
```

Comment:
Please make protected and fix tabulation one line 22. Change constructor to support those two new parameters and move default value to constructor.

```php
public function __construct($accessKey, $secretKey, $host = self::AWS_US_EAST_1, $verifyPeer = true, $verifyHost = true, LoggerInterface $logger = null)
{
    // ...
}
```